config/initializers/_constants.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```

Comment:
The file's name starts with an underscore to ensure the constants are loaded first

Reply:
Not sure if it is the best way (or at least the rails way) to centralize this kind of information